src/main/java/org/broadinstitute/hellbender/tools/copynumber/gcnv/GenerateVCFFromPosteriors.java

```diff
@@ -0,0 +1,224 @@
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+/**
+ * Create a VCF given the output of gCNV caller
+ *
+ * <p> This tool takes a list of directories of chunked gCNV call directories containing posterior records for
+ * different copy number states and outputs a VCF. Note that the list should be sorted according order of the intervals
+ * in corresponding chunks. The VCF will contain an ALT allele for every non-reference copy
+ * number state specified in the posterior files.
+ * </p>
+ *
+ * <h3>Input</h3>
+ * <ul>
+ *     <li>A list of paths to gCNV chunked calls directories</li>
+ *     <li>Name of the sample directory (it must be the same across all chunks)</li>
+ * </ul>
+ *
+ * <h3>Output</h3>
+ * <ul>
+ *     <li>A VCF file with CNV calls</li>
+ * </ul>
+ *
+ * <h3>Example</h3>
+ *
+ * <pre>
+ *   gatk GenerateVCFFromPosteriors \
+ *     --called-chunk-directory path/to/chunk_1
+ *     --called-chunk-directory path/to/chunk_2
+ *     --sample-name-directory SAMPLE_1
+ *     --output output.vcf
+ * </pre>
+ */
+@CommandLineProgramProperties(
+        summary = "Create a VCF given the output of gCNV caller",
+        oneLineSummary = "Create a VCF given the output of gCNV caller",
+        programGroup = CopyNumberProgramGroup.class
+)
+public class GenerateVCFFromPosteriors extends GATKTool {
+    private static final Logger logger = LogManager.getLogger(GenerateVCFFromPosteriors.class);
+
+    public static final String POSTERIOR_CALL_DIRECTORY_FULL_NAME = "called-chunk-directory";
+    public static final String POSTERIOR_CALL_DIRECTORY_SHORT_NAME = "CCD";
+
+    public static final String SAMPLE_DIRECTORY_NAME_FULL_NAME = "sample-name-directory";
+    public static final String SAMPLE_DIRECTORY_NAME_SHORT_NAME = "SN";
+
+    private static final String COMMENT_PREFIX = "@";
+
+    @Argument(
+            doc = "List of paths to chunk directories in a sorted order",
+            fullName = POSTERIOR_CALL_DIRECTORY_FULL_NAME,
+            shortName = POSTERIOR_CALL_DIRECTORY_SHORT_NAME
+    )
+    private List<File> orderedChunkDirectoryList = null;
+
+    @Argument(
+            doc = "Name of the sample directory in the gCNV called chunks (it must be the same across all chunks)",
+            fullName = SAMPLE_DIRECTORY_NAME_FULL_NAME,
+            shortName = SAMPLE_DIRECTORY_NAME_SHORT_NAME
+    )
+    private String sampleDirectoryName = null;
+
+    @Argument(
+            doc = "Output VCF file",
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME
+    )
+    private File outputFile = null;
+
+    /**
+     * Sample name derived from the copy number posterior file
+     */
+    private String sampleName;
+
+    final String variantPrefix = "CNV";
+
+    @Override
+    public void traverse() {
+        if (orderedChunkDirectoryList.size() <= 0) {
```

Comment:
Just use `minElements = 1` in the argument annotation.

Reply:
Done